AppCenter.podspec

```diff
@@ -38,19 +38,23 @@ Pod::Spec.new do |s|
 
   s.ios.deployment_target = '9.0'
   s.osx.deployment_target = '10.9'
+  s.tvos.deployment_target = '10.0'
```

Comment:
@jaelim-ms @clpolet thoughts?

Reply:
We can revisit when we'll have recent tvOS distribution numbers.

Reply:
Now we have a decision from @elamalani it's min tvOS 11. Please update.

Reply:
Actually I'll create another work item for just that because we have to update all the tvOS projects. Cleaner to do it in another work item.